org.eclipse.xtext.xbase/src/org/eclipse/xtext/xbase/controlflow/ConstantConditionsInterpreter.xtend

```diff
@@ -148,6 +148,7 @@ class ConstantConditionsInterpreter {
 								JvmIdentifiableElement,
 								ThisReference: <Object>newArrayList(v)
 								List<Object>: new ArrayList(v)
+								default: emptyList
```

Comment:
am not sure if a immutable is a good idea here. @szarnekow what do you think?

Reply:
@szarnekow ping

Reply:
I'll fix this sometime next week